git-repo-prep/static/book.md

```diff
@@ -0,0 +1,11 @@
+# {book_title}
```

Comment:
Could the README have a Gitpod badge?

Reply:
If you mean like this:
[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/openstax/osbooks-business-law)
It should not be too difficult to add.

Reply:
@TylerZeroMaster I think you can use the referrer URL inside the README instead of templating the repo name